src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -19,6 +19,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <CppLibCreator>lib</CppLibCreator>
     <FullRuntimeName>Runtime</FullRuntimeName>
     <FullRuntimeName Condition="'$(ServerGarbageCollection)' != ''">Runtime.ServerGC</FullRuntimeName>
+    <FullRuntimeName Condition="'$(ControlFlowGuard)' == 'Guard'">Runtime.ServerGC.GuardCF</FullRuntimeName>
```

Comment:
Do we also need GuardCF version of the corefx .libs (globalization, compression)?

Reply:
(It can be follow up PR.)

Reply:
Ah yes. We also need it for the bootstrapper.